src/main/java/codesquad/springcafe/article/Article.java

```diff
@@ -0,0 +1,31 @@
+package codesquad.springcafe.article;
+
+import java.time.LocalDateTime;
+
+public class Article {
+    private int id;
```

Comment:
```suggestion
    private Long id;
```

딱히 이유가 있는 건 아닌데 Long 타입 래퍼를 아이디로 선호하는 편입니다.

Reply:
잠깐 생각해 보니 아마 Hibernate의 영향인 것 같네요.